redux/adminSlice.ts

```diff
@@ -6,6 +6,8 @@ interface AdminState {
   isUsernameTakenError: boolean;
   isUsernameInvalidError: boolean;
   newBotToken: string;
+  isWaitingForBotsNames: boolean;
+  botsNames: string[];
```

Comment:
You can return to your original solution, it was better than adding a new variable